test/suite/test_timestamp03.py

```diff
@@ -235,7 +220,7 @@ def test_timestamp03(self):
             # Tables using the timestamps should see the values as per the
             # given read_timestamp
             self.check(self.session, 'read_timestamp=' + self.timestamp_str(t),
-                self.table_ts_log, dict((k, self.value) for k in orig_keys[:i+1]))
+                self.table_ts_log, dict((k, self.value) for k in orig_keys))
             self.check(self.session, 'read_timestamp=' + self.timestamp_str(t),
```

Comment:
I think this might want an explicit note pointing out that the ts_log table doesn't actually use timestamps, since the logic is already confusing without that wrinkle. :-|

Reply:
Fixed in a7e9a3cb9.